statsmodels/survey/summary_stats.py

```diff
@@ -0,0 +1,480 @@
+"""
+Methods for creating summary statistics and their SE for survey data.
+
+The main classes are:
```

Comment:
I wouldn't enumerate the classes here.  It just creates a parallel set of documentation that will likely become inconsistent with the real docstrings over time.  The purpose of the module-level docstring is to give an overall statement about the goals of the module, and often to provide some high-level references.